x-pack/plugins/ml/public/alerting/jobs_health_rule/register_jobs_health_alerting_rule.ts

```diff
@@ -61,7 +61,14 @@ export function registerJobsHealthAlertingRule(
       {
         defaultMessage: `Anomaly detection jobs health check result:
 \\{\\{context.message\\}\\}
-- Job IDs: \\{\\{context.jobIds\\}\\}
+\\{\\{#context.results\\}\\}
+  Job ID: \\{\\{job_id\\}\\}
+  \\{\\{#datafeed_id\\}\\}Datafeed ID: \\{\\{datafeed_id\\}\\}  \\{\\{/datafeed_id\\}\\}
+  \\{\\{#datafeed_state\\}\\}Datafeed state: \\{\\{datafeed_state\\}\\}  \\{\\{/datafeed_state\\}\\}
+  \\{\\{#memory_status\\}\\}Memory status: \\{\\{memory_status\\}\\}  \\{\\{/memory_status\\}\\}
+  \\{\\{#log_time\\}\\}Memory logging time: \\{\\{log_time\\}\\}  \\{\\{/log_time\\}\\}
+  \\{\\{#failed_category_count\\}\\}Failed category count: \\{\\{failed_category_count\\}\\}  \\{\\{/failed_category_count\\}\\}
+\\{\\{/context.results\\}\\}
```

Comment:
Maybe we should also include some text with possible action the user can take to remedy the issue e.g. assign the job more memory or edit the datafeed filter to limit scope of analysis if the soft limit has been reached, and assign the jobre more memory and restore from a snapshot from prior to reaching the hard limit.

Reply:
Yes, good suggestion, but the message is dynamic so it should be set server-side in the executor code. I will update it

Reply:
Added in https://github.com/elastic/kibana/pull/106763/commits/01bc9185686e4ef4ef610a299aa5f4a3764f34ee